packages/components/addon/components/hds/application-state/header.hbs

```diff
@@ -0,0 +1,19 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+<div class="hds-application-state__header" ...attributes>
+  {{#if @subtitle}}
+    <div class="hds-application-state__header--icon">
```

Comment:
If we want to follow traditional BEM format for these child element class names then the the extra dashes should be removed:

"hds-application-state__header--icon"
=>
"hds-application-state__header-icon"

Two dashes is normally used for a modifier of another class, for example:
```
<div class="header header--large">Blah</div>
```

Reply:
ok I think it's correct now 👍